workloads/templates/vm-creation.md

```diff
@@ -2,68 +2,109 @@
 
 ## Overview
 
-The [OpenShift's template mechanism](https://docs.openshift.org/latest/dev_guide/templates.html) allows user to create a set of objects from a template.  KubeVirt takes benefit from this template mechanism to create VirtualMachines.
+The KubeVirt projects provides a set of [templates](https://docs.okd.io/latest/dev_guide/templates.html) to create VMs to handle common usage scenarios.
+These templates provide a combination of some key factors that could be further customized and processed to have a Virtual Machine object.
+
+The key factors which define a template are
+ - Workload
+  Most Virtual Machine should be *generic* to have maximum flexibility; the *highperformance* workload trades some of this flexibility to
+  provide better performances.
+- Guest Operating System (OS)
+  This allow to ensure that the emulated hardware is compatible with the guest OS. Furthermore, it allows to maximize the stability
+  of the VM, and allows performance optimizations.
+- Size (flavor) 
+  Defines the amount of resources (CPU, memory) to allocate to the VM.
 
-In order to create a virtual machine via OpenShift CLI, you need to provide a template defining the corresponding object and its metadata.
 
-!> Only `VirtualMachine` object is currently supported.
+## WebUI
+Kubevirt project has [the official UI](https://github.com/kubevirt/web-ui). This UI supports creation VM using templates and templates features - flavors and workload profiles.
+To create VM from template, choose WorkLoads in the left panel >> press to the "Create Virtual Machine" blue button >> choose "Create from Wizzard". Next, you have to see "Create Virtual Machine" window
 
+## common-templates
+There is the [common-templates subproject](https://github.com/kubevirt/common-templates/) subproject. It provides official prepaired and useful templates. [Additional doc available](/workloads/templates/common-templates.md). You can also create templates by hand. You can find an example below, in the "Example template" section.
```

Comment:
```suggestion
Kubevirt provides a set of tested useful templates via the [common-templates](https://github.com/kubevirt/common-templates/) subproject. Those templates also contain additional metadata that provide more information to the WebUI that can then offer improved selection and validation logic. [Additional doc available](/workloads/templates/common-templates.md).

We recommend using a provided template, however you can create custom templates by hand too if needed. Please find an example below, in the "Example template" section.
```

Reply:
I've added it in the latest commit @MarSik